### PR TITLE
docs(ivy): update status.md

### PR DESCRIPTION
**Pull request by mhevery**

### CODE REVIEW

File: packages/core/src/render3/STATUS.md [hunk 1]
```diff
@@ -201,41 +182,41 @@ The goal is for the `@Component` (and friends) to be the compiler of template. S
...
-| sanitization with no injector       |  ✅     |  ✅      |  ❌      |
+| default `inject()` with no injector |  ✅     |  ✅      |  ✅      |
+| sanitization with no injector       |  ✅     |  ✅      |  ✅      |
 
 
 ### I18N
 | Feature                             | Runtime | Spec     | Compiler |
 | ----------------------------------- | ------- | -------- | -------- |
-| translate text literals             |  ✅     |  ✅      |  ✅      |
+| translate text literals             |  ❌     |  ❌      |  ✅      |
```
Comment (alfaproject) [inline packages/core/src/render3/STATUS.md]: ):

File: packages/core/src/render3/STATUS.md [hunk 2]
```diff
@@ -147,16 +132,12 @@ The goal is for the `@Component` (and friends) to be the compiler of template. S
...
+| `<div xmlns:foo="url" foo:bar="baz">` <br/>Compiler still needs to be updated to process templates with namespaced attributes. ([see #24386](https://github.com/angular/angular/pull/24386))       |  ✅     |  ✅      |  ✅      |
 | `{{ ['literal', exp ] }}`                   |  ✅     |  ✅      |  ✅      |
 | `{{ { a: 'literal', b: exp } }}`            |  ✅     |  ✅      |  ✅      |
 | `{{ exp \| pipe: arg }}`                    |  ✅     |  ✅      |  ✅      |
 | `<svg:g svg:p>`                             |  ✅     |  ✅      |  ✅      |
-| `<img src=[userData]>` sanitization         |  ❌     |  ❌      |  ❌      |
-| `<div (nocd.click)>`                        |  ❌     |  ❌      |  ❌      |
-| `<div (bubble.click)>`                      |  ❌     |  ❌      |  ❌      |
-| `<div (keyup.enter)>`                       |  ❌     |  ❌      |  ❌      |
-| `<div (hammer.js)>`                         |  ❌     |  ❌      |  ❌      |
```
Comment (pkozlowski-opensource) [inline packages/core/src/render3/STATUS.md]: All the gestures with hammer.js work today so we should either preserve this or designed an alternative (hammer.js seems to be getting outdated and not maintained those days, if I'm not mistaken).

File: packages/core/src/render3/STATUS.md [hunk 3]
```diff
@@ -147,16 +132,12 @@ The goal is for the `@Component` (and friends) to be the compiler of template. S
 | `<div style="literal">`                     |  ✅     |  ✅      |  ✅      |
 | `<div [style]="exp">`                       |  ✅     |  ✅      |  ✅      |
 | `<div [style.foo]="exp">`                   |  ✅     |  ✅      |  ✅      |
-| `<div xmlns:foo="url" foo:bar="baz">` <br/>Compiler still needs to be updated to process templates with namespaced attributes. ([see #24386](https://github.com/angular/angular/pull/24386))       |  ✅     |  ✅      |  ❌      |
+| `<div xmlns:foo="url" foo:bar="baz">` <br/>Compiler still needs to be updated to process templates with namespaced attributes. ([see #24386](https://github.com/angular/angular/pull/24386))       |  ✅     |  ✅      |  ✅      |
```
[the remark this answers is not in the document]
Comment (mhevery) [reply packages/core/src/render3/STATUS.md]: good point deleted.

File: packages/core/src/render3/STATUS.md [hunk 4]
```diff
@@ -201,41 +182,41 @@ The goal is for the `@Component` (and friends) to be the compiler of template. S
 | ------------------------------- | ------- | -------- | -------- |
 | `<ng-content>`                  |  ✅     |  ✅      |  ✅      |
 | `<ng-content selector="...">`   |  ✅     |  ✅      |  ✅      |
-| container `projectAs`           |  ✅     |  ✅      |  ❌      |
+| container `ngProjectAs`         |  ✅     |  ✅      |  ❌      |
```
[the remark this answers is not in the document]
Comment (mhevery) [reply packages/core/src/render3/STATUS.md]: Yes, updated

[the remark this answers is not in the document]
Comment (mhevery) [reply packages/core/src/render3/STATUS.md]: updated

File: packages/core/src/render3/STATUS.md [hunk 5]
```diff
@@ -147,16 +132,12 @@ The goal is for the `@Component` (and friends) to be the compiler of template. S
...
-| `<div xmlns:foo="url" foo:bar="baz">` <br/>Compiler still needs to be updated to process templates with namespaced attributes. ([see #24386](https://github.com/angular/angular/pull/24386))       |  ✅     |  ✅      |  ❌      |
+| `<div xmlns:foo="url" foo:bar="baz">` <br/>Compiler still needs to be updated to process templates with namespaced attributes. ([see #24386](https://github.com/angular/angular/pull/24386))       |  ✅     |  ✅      |  ✅      |
 | `{{ ['literal', exp ] }}`                   |  ✅     |  ✅      |  ✅      |
 | `{{ { a: 'literal', b: exp } }}`            |  ✅     |  ✅      |  ✅      |
 | `{{ exp \| pipe: arg }}`                    |  ✅     |  ✅      |  ✅      |
 | `<svg:g svg:p>`                             |  ✅     |  ✅      |  ✅      |
-| `<img src=[userData]>` sanitization         |  ❌     |  ❌      |  ❌      |
-| `<div (nocd.click)>`                        |  ❌     |  ❌      |  ❌      |
-| `<div (bubble.click)>`                      |  ❌     |  ❌      |  ❌      |
-| `<div (keyup.enter)>`                       |  ❌     |  ❌      |  ❌      |
```
[the remark this answers is not in the document]
Comment (mhevery) [reply packages/core/src/render3/STATUS.md]: Yes, but it is all existing functionality from `Renderer2` and so I don't think we need to track it here.

File: packages/core/src/render3/STATUS.md (hunk 2 above)
Comment (mhevery) [reply packages/core/src/render3/STATUS.md]: same as above comment.

File: packages/core/src/render3/STATUS.md (hunk 1 above)
Comment (mhevery) [reply packages/core/src/render3/STATUS.md]: We realized that the current implementation was flawed. Should have a fix soon.

**Review by pkozlowski-opensource — APPROVED** [review]
LGTM with the latest fixup.

File: packages/core/src/render3/STATUS.md [hunk 6]
```diff
@@ -201,41 +182,41 @@ The goal is for the `@Component` (and friends) to be the compiler of template. S
...
-| `directiveInject(Type)`             |  ✅     |  ✅      |  ❌      |
+| `directiveInject(Type)`             |  ✅     |  ✅      |  ✅      |
 | `inject(Type, SkipSelf)`            |  ❌     |  ❌      |  ❌      |
 | `attribute('name')`                 |  ✅     |  ✅      |  ❌      |
-| `injectChangeDetectionRef()`        |  ✅     |  ✅      |  ❌      |
+| `injectChangeDetectionRef()`        |  ✅     |  ✅      |  ✅      |
 | `injectElementRef()`                |  ✅     |  ✅      |  ✅      |
 | `injectViewContainerRef()`          |  ✅     |  ✅      |  ✅      |
 | `injectTemplateRef()`               |  ✅     |  ✅      |  ✅      |
 | `injectRenderer2()`                 |  ✅     |  ✅      |  ✅      |
```
Comment (trotyl) [inline packages/core/src/render3/STATUS.md]: Wasn't these specific instruction being removed at https://github.com/angular/angular/pull/26064 and https://github.com/angular/angular/pull/26369?

File: packages/core/src/render3/STATUS.md [hunk 7]
```diff
@@ -201,41 +182,41 @@ The goal is for the `@Component` (and friends) to be the compiler of template. S
...
 
 
 ### Injection Features
 | Feature                             | Runtime | Spec     | Compiler |
 | ----------------------------------- | ------- | -------- | -------- |
 | `inject(Type)`                      |  ✅     |  ✅      |  ✅      |
-| `directiveInject(Type)`             |  ✅     |  ✅      |  ❌      |
+| `directiveInject(Type)`             |  ✅     |  ✅      |  ✅      |
 | `inject(Type, SkipSelf)`            |  ❌     |  ❌      |  ❌      |
 | `attribute('name')`                 |  ✅     |  ✅      |  ❌      |
```
Comment (trotyl) [inline packages/core/src/render3/STATUS.md]: Isn't it `injectAttribute()`?

**Pull request closed without merging**